client/dashboard/data/types.ts

```diff
@@ -92,8 +92,10 @@ export interface Site {
 	is_wpcom_staging_site: boolean;
 	launch_status: string | boolean;
 	site_migration: {
-		migration_status: string;
-	} | null;
+		migration_status?: string;
+		in_progress: boolean;
+		is_complete: boolean;
+	};
```

Comment:
Why did you remove `| null`? It can still be null?

Reply:
I assumed that we're passing site_migration to the API field so it will always return it. Is it not the case?

Reply:
Oh, has the API changed? I could swear I saw `null` before. In that case, we should be able to remove the optional chaining for `site_migration`.